server/src/test/java/io/crate/expression/predicate/FieldExistsQueryTest.java

```diff
@@ -102,10 +102,9 @@ private void assertNotFunction(String createStatement, Object[] values) throws E
             if (values == null) {
                 assertThat(results).isEmpty();
             } else {
-                assertThat(results).hasSize(2);
+                assertThat(results).hasSize(1);
```

Comment:
Please double check this change, I think it's correct, and tested it end to end with:
```
cr> insert into t1 values ('{}');
INSERT OK, 1 row affected (0.035 sec)
cr> refresh table t1;
REFRESH OK, 1 row affected (0.003 sec)
cr> SELECT * FROM t1 WHERE (((CAST(t1.c0 AS VARCHAR))<>('')));
+----+
| c0 |
+----+
| {} |
+----+
```